src/app/phast/losses/wall-losses/wall-losses.component.html

```diff
@@ -29,10 +29,10 @@ <h4>To add Wall Losses you will need to delete your current modifications and re
     <!--wall losses form with data injected-->
     <app-wall-losses-form (changeField)="changeField($event)" [(wallLossesForm)]="loss.form"
       (calculate)="calculate(loss)" [baselineSelected]="baselineSelected" (saveEmit)="saveLosses()"
-      [lossIndex]="lossIndex" [settings]="settings" (inputError)="setError($event)" [inSetup]="inSetup"
+      [lossIndex]="lossIndex" [settings]="settings" [inSetup]="inSetup"
       [isBaseline]="isBaseline"></app-wall-losses-form>
   </div>
-  <div class="loss-results" [ngClass]="{'input-error': showError == true}">
```

Comment:
looking at this I do think we want to indicate there is an error here. Unless the heatLoss is not being calculated at all when the form is invalid. We should double check this behavior is working correctly.